com.unity.multiplayer.mlapi/Runtime/com.unity.multiplayer.mlapi.runtime.asmdef

```diff
@@ -1,14 +1,23 @@
 {
     "name": "Unity.Multiplayer.MLAPI.Runtime",
     "rootNamespace": "MLAPI",
-    "references": [],
+    "references": [
+        "GUID:3450dc9e228065a499e6e09dee931704",
+        "GUID:b1cd7326e664a434ab35daa802773c7f"
```

Comment:
Looks like we're pulling a reference to NetworkProfiler, lets see if we can remove that dependency